lib/modules/manager/gomod/__snapshots__/extract.spec.ts.snap

```diff
@@ -605,10 +605,12 @@ exports[`modules/manager/gomod/extract extractPackageFile() extracts multi-line
     },
   },
   {
+    "currentDigest": "43d17e14b714",
```

Comment:
this test was amended to catch the fact that digests are now always extracted for packages that are added with a digest value instead of just a version tag